builder.go

```diff
@@ -12,8 +12,8 @@ import (
 var _ ipld.NodePrototype = (*Prototype)(nil)
 
 type Prototype struct {
-	BitWidth   int
-	BucketSize int
+	BitWidth   int64
```

Comment:
This change makes me wonder, do we have anything in this codebase that places limits on these values? It might be an easy DoS or cause of panic to provide a block with unreasonable values for these, there should be some pretty low limits for both of them (`BucketSize` would be a little more arbitrary than `BitWidth`).

Reply:
The rationale I had was `int` in IPLD schema is represented as `int64` so why not use the matching type in go. 

Happy to revert this back to `int` and take the discussion further over a separate issue.

Reply:
OK reverting this back to int after reading though the code comments; thank you for pointing this out 🙇